app/models/card.rb

```diff
@@ -9,6 +18,6 @@ def set_review_date
     end
 
     def fields_are_not_equal
-      self.errors.add(:base, 'Original text and translated text cannot be equal.') if self.original_text == self.translated_text
+      errors.add(:base, 'Original text and translated text cannot be equal.') if original_text == translated_text
```

Comment:
Line is too long. [113/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.